src/PrestaShopBundle/Form/Validator/Constraints/TinyMceMaxLength.php

```diff
@@ -47,4 +51,9 @@ public function __construct($options = null)
             throw new MissingOptionsException(sprintf('Option "max" must be given for constraint %s', __CLASS__), ['max']);
         }
     }
+
+    public function validatedBy()
```

Comment:
```suggestion
    public function validatedBy(): string
```

Reply:
defined by interface, I'm not sure we can do it https://github.com/symfony/validator/blob/3.4/ConstraintValidatorInterface.php#L34

Reply:
`validate` is not `validatedBy` ;)

Reply:
sorry alcohol effect !